source/pip/src/interpreter.rs

```diff
@@ -1303,23 +1304,27 @@ pub(crate) struct CircuitConfig {
     pub(crate) source_locations: Option<bool>,
     #[pyo3(get, set)]
     pub(crate) group_by_scope: Option<bool>,
+    #[pyo3(get, set)]
+    pub(crate) prune_classical_qubits: bool,
 }
 
 #[pymethods]
 impl CircuitConfig {
     #[new]
-    #[pyo3(signature=(*,max_operations=None, generation_method=None, source_locations=None, group_by_scope=None))]
+    #[pyo3(signature=(*,max_operations=None, generation_method=None, source_locations=None, group_by_scope=None, prune_classical_qubits=false))]
     fn new(
         max_operations: Option<usize>,
         generation_method: Option<CircuitGenerationMethod>,
         source_locations: Option<bool>,
         group_by_scope: Option<bool>,
+        prune_classical_qubits: bool,
```

Comment:
I can't process why this flag looks different than the others. Why not `Option<bool>`? Should the others have been `bool`?